src/rwe/GameScene.h

```diff
@@ -231,9 +231,44 @@ namespace rwe
         bool up{false};
         bool down{false};
 
+        bool leftCtrlDown{false};
+        bool rightCtrlDown{false};
         bool leftShiftDown{false};
         bool rightShiftDown{false};
 
+        bool trackingOn{false};
+        UnitId trackedUnitId;
+
+        void StartTrack()
+        {
+            if (selectedUnits.size())
+            {
+                // sort selection by unit id so repeated 'T' keydown cycles through all units in a group consistently
+                std::vector<UnitId> unitIds;
+                for (const auto& u : selectedUnits)
+                {
+                    unitIds.push_back(u);
+                }
+                sort(unitIds.begin(), unitIds.end());
+                
+                // If already tracking, check if currently tracked unit is in this selection. If it is, select the next id in the group.
+                auto it = find(unitIds.begin(), unitIds.end(), trackedUnitId);
+                if (trackingOn && it != unitIds.end() && ++it != unitIds.end())
+                {
+                    trackedUnitId = *it;
+                }
+                else
+                {
+                    trackedUnitId = unitIds[0];
+                }
+                trackingOn = true;
+            }
+            else
+            {
+                trackingOn = false;
+            }
```

Comment:
So the trackingOn variable is needed to stop tracking when it's supposed to stop... but yeah I think technically this else just turns off tracking if the unit is dead/doesn't exist anymore, in which case tryGetUnit and isDead will already prevent the camera from trying to move. So all it really does is replace those checks with a bool check each update after a unit goes away.
I'll get rid of it as the benefit is questionable and it's simpler. Plus it might even need to go away when projectile tracking is implemented.